utils/navigator_gui/navigator_gui/dashboard.py

```diff
@@ -156,8 +154,7 @@ def connect_ros(self):
         rospy.Subscriber("/battery_monitor", Float32, self.cache_battery_voltage)
         rospy.Subscriber("/clock", Clock, self.cache_system_time)
 
-        self.wrench_changer = rospy.ServiceProxy('/change_wrench', WrenchSelect)
```

Comment:
After the actual wrench changing was moved to the remote control library and the service call method of checking it's status was switched to caching messages from a publisher, this was an unused service client.